variants/devise/spec/factories/users.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :user do
+    email { "user-#{SecureRandom.hex(5)}@example.com" }
+
+    # User#password is a virtual attribute (i.e. it is not stored direclty in
+    # the DB) so we need call the User#password= method after building the user
+    # to set it.
+    after(:build) { |u| u.password = "aabbccdd" }
```

Comment:
I've always been able to just do `password { "aabbccdd" }; password_confirmation { password }` here to have this work without the `after` hook

Reply:
hmmm I want to say that didn't work for me but I'm probably wrong. Will try it.